web/themes/custom/pineapple/sass/components/paragraphs/_image-grid.scss

```diff
@@ -0,0 +1,50 @@
+.field--name-field-p-ig-item {
+  @include breakpoint($lap) {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .field__item {
```

Comment:
Same for this as this: https://github.com/katlazarova/easy-recipe-delight/pull/44/files#r373775768

Reply:
okay.